diff --git a/node_modules/@solana/wallet-adapter-react-ui/src/WalletMultiButton.tsx b/node_modules/@solana/wallet-adapter-react-ui/src/WalletMultiButton.tsx
index 2332bbc..83a511c 100644
--- a/node_modules/@solana/wallet-adapter-react-ui/src/WalletMultiButton.tsx
+++ b/node_modules/@solana/wallet-adapter-react-ui/src/WalletMultiButton.tsx
@@ -71,7 +71,7 @@ export const WalletMultiButton: FC<ButtonProps> = ({ children, ...props }) => {
             >
                 {content}
             </Button>
-            <ul
+            {/* <ul
                 aria-label="dropdown-list"
                 className={`wallet-adapter-dropdown-list ${active && 'wallet-adapter-dropdown-list-active'}`}
                 ref={ref}
@@ -86,7 +86,7 @@ export const WalletMultiButton: FC<ButtonProps> = ({ children, ...props }) => {
                 <li onClick={disconnect} className="wallet-adapter-dropdown-list-item" role="menuitem">
                     Disconnect
                 </li>
-            </ul>
+            </ul> */}
         </div>
     );
 };
