diff --git a/node_modules/@solana/wallet-adapter-react-ui/src/WalletMultiButton.tsx b/node_modules/@solana/wallet-adapter-react-ui/src/WalletMultiButton.tsx
index 2332bbc..101e0a7 100644
--- a/node_modules/@solana/wallet-adapter-react-ui/src/WalletMultiButton.tsx
+++ b/node_modules/@solana/wallet-adapter-react-ui/src/WalletMultiButton.tsx
@@ -77,15 +77,6 @@ export const WalletMultiButton: FC<ButtonProps> = ({ children, ...props }) => {
                 ref={ref}
                 role="menu"
             >    
-                <li onClick={copyAddress} className="wallet-adapter-dropdown-list-item" role="menuitem">
-                    {copied ? 'Copied' : 'Copy address'}
-                </li>
-                <li onClick={openModal} className="wallet-adapter-dropdown-list-item" role="menuitem">
-                    Connect a different wallet
-                </li>
-                <li onClick={disconnect} className="wallet-adapter-dropdown-list-item" role="menuitem">
-                    Disconnect
-                </li>
             </ul>
         </div>
     );
